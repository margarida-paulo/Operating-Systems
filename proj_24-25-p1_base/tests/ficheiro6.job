WRITE [(Liam,Nelson)(Emma,Stone)]
WRITE [(Chris,Hemsworth)(Scarlett,Johansson)]
WRITE [(Robert,Downey)(Natalie,Portman)]
WRITE [(Leonardo,DiCaprio)(Kate,Winslet)]
WRITE [(Denzel,Washington)(Meryl,Streep)]
WRITE [(Tom,Hanks)(Julia,Roberts)]
WRITE [(Will,Smith)(Margot,Robbie)]
WRITE [(Christian,Bale)(Amy,Adams)]
WRITE [(Brad,Pitt)(Angelina,Jolie)]
WRITE [(Johnny,Depp)(Helena,Bonham)]
WRITE [(Morgan,Freeman)(Anne,Hathaway)]
WRITE [(Matt,Damon)(Sandra,Bullock)]
WRITE [(Joaquin,Phoenix)(Reese,Witherspoon)]
WRITE [(Daniel,Radcliffe)(Emma,Watson)]
WRITE [(Hugh,Jackman)(Nicole,Kidman)]
WRITE [(Ryan,Gosling)(Rachel,McAdams)]
WRITE [(Keanu,Reeves)(Carrie,Moss)]
WRITE [(Chadwick,Boseman)(Lupita,Nyong)]
WRITE [(Ben,Affleck)(Gal,Gadot)]
WRITE [(Jason,Momoa)(Amber,Heard)]

