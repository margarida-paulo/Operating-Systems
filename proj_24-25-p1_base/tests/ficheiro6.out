(Ana, Martins)
(Alexandre, Rodrigues)
(Adriana, Lopes)
(Alberto, Pereira)
(Albert, Camus)
(Aldous, Huxley)
(Agatha, Christie)
(Chris, Hemsworth)
(Clara, Sousa)
(Cristina, Sousa)
(Carlos, Vinícius)
(Chinua, Achebe)
(C.S., Lewis)
(Charles, Dickens)
(Clarice, Lispector)
(Denzel, Washington)
(Emma, Stone)
(Ernest, Hemingway)
(Edgar, Allan)
(Emily, Dickinson)
(Fabiana, Alves)
(Fyodor, Dostoevsky)
(Franz, Kafka)
(Fernando, Pessoa)
(Gustavo, Lima)
(George, Orwell)
(Gabriel, Marquez)
(Haruki, Murakami)
(Herman, Melville)
(Isabel, Allende)
(Italo, Calvino)
(Julia, Roberts)
(João, Pinto)
(James, Joyce)
(J.R.R., Tolkien)
(J.K., Rowling)
(Jane, Austen)
(José, Saramago)
(Jorge, Luis)
(Kate, Winslet)
(Leonardo, DiCaprio)
(Liam, Nelson)
(Leo, Tolstoy)
(Luís, Carvalho)
(Margot, Robbie)
(Meryl, Streep)
(Marta, Ribeiro)
(Maria, Costa)
(Margarida, Maggie)
(Mariana, Mari)
(Marcel, Proust)
(Mark, Twain)
(Miguel, Cervantes)
(Natalie, Portman)
(Nicolas, Mendes)
(Oscar, Wilde)
(Pedro, Santos)
(Paulo, Carvalho)
(Pablo, Neruda)
(Robert, Downey)
(Ricardo, Oliveira)
(Renata, Teixeira)
(Scarlett, Johansson)
(Simone, Oliveira)
(Stephen, King)
(Sophia, Andresen)
(Tom, Hanks)
(Toni, Morrison)
(Victor, Hugo)
(Virginia, Woolf)
(Will, Smith)
(William, Faulkner)
(Zadie, Smith)
(Ana, Martins)
(Alexandre, Rodrigues)
(Adriana, Lopes)
(Alberto, Pereira)
(Albert, Camus)
(Aldous, Huxley)
(Agatha, Christie)
(Chris, Hemsworth)
(Clara, Sousa)
(Cristina, Sousa)
(Carlos, Vinícius)
(Chinua, Achebe)
(C.S., Lewis)
(Charles, Dickens)
(Clarice, Lispector)
(Denzel, Washington)
(Emma, Stone)
(Ernest, Hemingway)
(Edgar, Allan)
(Emily, Dickinson)
(Fabiana, Alves)
(Fyodor, Dostoevsky)
(Franz, Kafka)
(Fernando, Pessoa)
(Gustavo, Lima)
(George, Orwell)
(Gabriel, Marquez)
(Haruki, Murakami)
(Herman, Melville)
(Isabel, Allende)
(Italo, Calvino)
(Julia, Roberts)
(João, Pinto)
(James, Joyce)
(J.R.R., Tolkien)
(J.K., Rowling)
(Jane, Austen)
(José, Saramago)
(Jorge, Luis)
(Kate, Winslet)
(Leonardo, DiCaprio)
(Liam, Nelson)
(Leo, Tolstoy)
(Luís, Carvalho)
(Margot, Robbie)
(Meryl, Streep)
(Marta, Ribeiro)
(Maria, Costa)
(Margarida, Maggie)
(Mariana, Mari)
(Marcel, Proust)
(Mark, Twain)
(Miguel, Cervantes)
(Natalie, Portman)
(Nicolas, Mendes)
(Oscar, Wilde)
(Pedro, Santos)
(Paulo, Carvalho)
(Pablo, Neruda)
(Robert, Downey)
(Ricardo, Oliveira)
(Renata, Teixeira)
(Scarlett, Johansson)
(Simone, Oliveira)
(Stephen, King)
(Sophia, Andresen)
(Tom, Hanks)
(Toni, Morrison)
(Victor, Hugo)
(Virginia, Woolf)
(Will, Smith)
(William, Faulkner)
(Zadie, Smith)
(Anne, Hathaway)
(Angelina, Jolie)
(Amy, Adams)
(André, Monteiro)
(Ana, Martins)
(Alexandre, Rodrigues)
(Adriana, Lopes)
(Alberto, Pereira)
(Albert, Camus)
(Aldous, Huxley)
(Agatha, Christie)
(Brad, Pitt)
(Christian, Bale)
(Carla, Lopes)
(Chris, Hemsworth)
(Clara, Sousa)
(Cristina, Sousa)
(Carlos, Vinícius)
(Chinua, Achebe)
(C.S., Lewis)
(Charles, Dickens)
(Clarice, Lispector)
(Diana, Rocha)
(Denzel, Washington)
(Emma, Stone)
(Ernest, Hemingway)
(Edgar, Allan)
(Emily, Dickinson)
(Fabiana, Alves)
(Fyodor, Dostoevsky)
(Franz, Kafka)
(Fernando, Pessoa)
(Gonçalo, Cruz)
(Gustavo, Lima)
(George, Orwell)
(Gabriel, Marquez)
(Helena, Bonham)
(Haruki, Murakami)
(Herman, Melville)
(Inês, Mendes)
(Isabel, Allende)
(Italo, Calvino)
(Johnny, Depp)
(Julia, Roberts)
(João, Pinto)
(James, Joyce)
(J.R.R., Tolkien)
(J.K., Rowling)
(Jane, Austen)
(José, Saramago)
(Jorge, Luis)
(Kate, Winslet)
(Leonardo, DiCaprio)
(Liam, Nelson)
(Leo, Tolstoy)
(Luís, Carvalho)
(Morgan, Freeman)
(Marco, Ferreira)
(Margot, Robbie)
(Meryl, Streep)
(Marta, Ribeiro)
(Maria, Costa)
(Margarida, Maggie)
(Mariana, Mari)
(Marcel, Proust)
(Mark, Twain)
(Miguel, Cervantes)
(Natalie, Portman)
(Nicolas, Mendes)
(Oscar, Wilde)
(Patrícia, Silva)
(Pedro, Santos)
(Paulo, Carvalho)
(Pablo, Neruda)
(Rui, Nogueira)
(Robert, Downey)
(Ricardo, Oliveira)
(Renata, Teixeira)
(Scarlett, Johansson)
(Simone, Oliveira)
(Stephen, King)
(Sophia, Andresen)
(Tiago, Pereira)
(Tom, Hanks)
(Toni, Morrison)
(Victor, Hugo)
(Virginia, Woolf)
(Will, Smith)
(William, Faulkner)
(Zadie, Smith)
(Amanda, Santos)
(Alexandra, Barros)
(Anne, Hathaway)
(Angelina, Jolie)
(Amy, Adams)
(André, Monteiro)
(Ana, Martins)
(Alexandre, Rodrigues)
(Adriana, Lopes)
(Alberto, Pereira)
(Albert, Camus)
(Aldous, Huxley)
(Agatha, Christie)
(Brad, Pitt)
(Camila, Ribeiro)
(Christian, Bale)
(Carla, Lopes)
(Chris, Hemsworth)
(Clara, Sousa)
(Cristina, Sousa)
(Carlos, Ramos)
(Chinua, Achebe)
(C.S., Lewis)
(Charles, Dickens)
(Clarice, Lispector)
(Daniel, Radcliffe)
(Diana, Rocha)
(Denzel, Washington)
(Eduarda, Silva)
(Emma, Watson)
(Ernest, Hemingway)
(Edgar, Allan)
(Emily, Dickinson)
(Francisco, Ferreira)
(Fabiana, Alves)
(Fyodor, Dostoevsky)
(Franz, Kafka)
(Fernando, Gomes)
(Gonçalo, Cruz)
(Gustavo, Lima)
(George, Orwell)
(Gabriel, Marquez)
(Hugh, Jackman)
(Hugo, Pires)
(Helena, Bonham)
(Haruki, Murakami)
(Herman, Melville)
(Inês, Mendes)
(Isabel, Allende)
(Italo, Calvino)
(Joaquin, Phoenix)
(Johnny, Depp)
(Julia, Roberts)
(João, Pinto)
(James, Joyce)
(J.R.R., Tolkien)
(J.K., Rowling)
(Jane, Austen)
(José, Martins)
(Jorge, Luis)
(Kate, Winslet)
(Luciana, Nogueira)
(Laura, Gomes)
(Leonardo, DiCaprio)
(Liam, Nelson)
(Leo, Tolstoy)
(Luís, Carvalho)
(Matt, Damon)
(Morgan, Freeman)
(Marco, Ferreira)
(Margot, Robbie)
(Meryl, Streep)
(Marta, Ribeiro)
(Maria, Costa)
(Margarida, Maggie)
(Mariana, Amorim)
(Marcel, Proust)
(Mark, Twain)
(Miguel, Pires)
(Nicole, Kidman)
(Natalie, Portman)
(Nicolas, Mendes)
(Oscar, Wilde)
(Patrícia, Barros)
(Pedro, Santos)
(Paulo, Carvalho)
(Pablo, Neruda)
(Rachel, McAdams)
(Ryan, Gosling)
(Reese, Witherspoon)
(Rafael, Martins)
(Raquel, Moura)
(Rui, Nogueira)
(Robert, Downey)
(Ricardo, Mendes)
(Renata, Teixeira)
(Sandra, Bullock)
(Sérgio, Oliveira)
(Sara, Marques)
(Sofia, Monteiro)
(Scarlett, Johansson)
(Simone, Oliveira)
(Stephen, King)
(Sophia, Andresen)
(Tiago, Pereira)
(Tom, Hanks)
(Toni, Morrison)
(Victor, Hugo)
(Virginia, Woolf)
(Will, Smith)
(William, Faulkner)
(Zadie, Smith)
(Amber, Heard)
(Anderson, Moura)
(Amanda, Santos)
(Alexandra, Barros)
(Anne, Hathaway)
(Angelina, Jolie)
(Amy, Adams)
(André, Monteiro)
(Ana, Martins)
(Alexandre, Rodrigues)
(Adriana, Lopes)
(Alberto, Pereira)
(Albert, Camus)
(Aldous, Huxley)
(Agatha, Christie)
(Ben, Affleck)
(Brad, Pitt)
(Chadwick, Boseman)
(Carrie, Moss)
(Carolina, Gomes)
(Cláudia, Barbosa)
(Camila, Ribeiro)
(Christian, Bale)
(Carla, Lopes)
(Chris, Hemsworth)
(Clara, Sousa)
(Cristina, Sousa)
(Carlos, Ramos)
(Chinua, Achebe)
(C.S., Lewis)
(Charles, Dickens)
(Clarice, Lispector)
(Douglas, Ramos)
(Diego, Marques)
(Daniel, Radcliffe)
(Diana, Rocha)
(Denzel, Washington)
(Eduarda, Silva)
(Emma, Watson)
(Ernest, Hemingway)
(Edgar, Allan)
(Emily, Dickinson)
(Fábio, Castro)
(Francisco, Ferreira)
(Fabiana, Alves)
(Fyodor, Dostoevsky)
(Franz, Kafka)
(Fernando, Gomes)
(Gal, Gadot)
(Gisele, Ferreira)
(Gonçalo, Cruz)
(Gustavo, Lima)
(George, Orwell)
(Gabriel, Marquez)
(Hugh, Jackman)
(Hugo, Pires)
(Helena, Bonham)
(Haruki, Murakami)
(Herman, Melville)
(Inês, Mendes)
(Isabel, Allende)
(Italo, Calvino)
(Jason, Momoa)
(Júnior, Bastos)
(Juliano, Rocha)
(Jéssica, Costa)
(Joana, Almeida)
(Joaquin, Phoenix)
(Johnny, Depp)
(Julia, Roberts)
(João, Pinto)
(James, Joyce)
(J.R.R., Tolkien)
(J.K., Rowling)
(Jane, Austen)
(José, Martins)
(Jorge, Luis)
(Keanu, Reeves)
(Kelly, Neves)
(Kate, Winslet)
(Lupita, Nyong)
(Larissa, Amorim)
(Luciana, Nogueira)
(Laura, Gomes)
(Leonardo, DiCaprio)
(Liam, Nelson)
(Leo, Tolstoy)
(Luís, Carvalho)
(Márcio, Coelho)
(Matt, Damon)
(Morgan, Freeman)
(Marco, Ferreira)
(Margot, Robbie)
(Meryl, Streep)
(Marta, Ribeiro)
(Maria, Costa)
(Margarida, Maggie)
(Mariana, Matos)
(Marcel, Proust)
(Mark, Twain)
(Miguel, Pires)
(Nicole, Kidman)
(Natalie, Portman)
(Nicolas, Mendes)
(Oscar, Wilde)
(Patrícia, Barros)
(Pedro, Santos)
(Paulo, Carvalho)
(Pablo, Neruda)
(Rodrigo, Pinto)
(Rogério, Lima)
(Rachel, McAdams)
(Ryan, Gosling)
(Reese, Witherspoon)
(Rafael, Antunes)
(Raquel, Moura)
(Rui, Nogueira)
(Robert, Downey)
(Ricardo, Mendes)
(Renata, Teixeira)
(Sandra, Bullock)
(Sérgio, Oliveira)
(Sara, Marques)
(Sofia, Monteiro)
(Scarlett, Johansson)
(Simone, Oliveira)
(Stephen, King)
(Sophia, Andresen)
(Tatiana, Vieira)
(Tiago, Pereira)
(Tom, Hanks)
(Toni, Morrison)
(Viviane, Lima)
(Victor, Hugo)
(Virginia, Woolf)
(Will, Smith)
(William, Faulkner)
(Zadie, Smith)
(Amber, Heard)
(Anderson, Moura)
(Amanda, Santos)
(Alexandra, Barros)
(Anne, Hathaway)
(Angelina, Jolie)
(Amy, Adams)
(André, Monteiro)
(Ana, Martins)
(Alexandre, Rodrigues)
(Adriana, Lopes)
(Alberto, Pereira)
(Albert, Camus)
(Aldous, Huxley)
(Agatha, Christie)
(Ben, Affleck)
(Brad, Pitt)
(Chadwick, Boseman)
(Carrie, Moss)
(Carolina, Gomes)
(Cláudia, Barbosa)
(Camila, Ribeiro)
(Christian, Bale)
(Carla, Lopes)
(Chris, Hemsworth)
(Clara, Sousa)
(Cristina, Sousa)
(Carlos, Ramos)
(Chinua, Achebe)
(C.S., Lewis)
(Charles, Dickens)
(Clarice, Lispector)
(Douglas, Ramos)
(Diego, Marques)
(Daniel, Radcliffe)
(Diana, Rocha)
(Denzel, Washington)
(Eduarda, Silva)
(Emma, Watson)
(Ernest, Hemingway)
(Edgar, Allan)
(Emily, Dickinson)
(Fábio, Castro)
(Francisco, Ferreira)
(Fabiana, Alves)
(Fyodor, Dostoevsky)
(Franz, Kafka)
(Fernando, Gomes)
(Gal, Gadot)
(Gisele, Ferreira)
(Gonçalo, Cruz)
(Gustavo, Lima)
(George, Orwell)
(Gabriel, Marquez)
(Hugh, Jackman)
(Hugo, Pires)
(Helena, Bonham)
(Haruki, Murakami)
(Herman, Melville)
(Inês, Mendes)
(Isabel, Allende)
(Italo, Calvino)
(Jason, Momoa)
(Júnior, Bastos)
(Juliano, Rocha)
(Jéssica, Costa)
(Joana, Almeida)
(Joaquin, Phoenix)
(Johnny, Depp)
(Julia, Roberts)
(João, Pinto)
(James, Joyce)
(J.R.R., Tolkien)
(J.K., Rowling)
(Jane, Austen)
(José, Martins)
(Jorge, Luis)
(Keanu, Reeves)
(Kelly, Neves)
(Kate, Winslet)
(Lupita, Nyong)
(Larissa, Amorim)
(Luciana, Nogueira)
(Laura, Gomes)
(Leonardo, DiCaprio)
(Liam, Nelson)
(Leo, Tolstoy)
(Luís, Carvalho)
(Márcio, Coelho)
(Matt, Damon)
(Morgan, Freeman)
(Marco, Ferreira)
(Margot, Robbie)
(Meryl, Streep)
(Marta, Ribeiro)
(Maria, Costa)
(Margarida, Maggie)
(Mariana, Matos)
(Marcel, Proust)
(Mark, Twain)
(Miguel, Pires)
(Nicole, Kidman)
(Natalie, Portman)
(Nicolas, Mendes)
(Oscar, Wilde)
(Patrícia, Barros)
(Pedro, Santos)
(Paulo, Carvalho)
(Pablo, Neruda)
(Rodrigo, Pinto)
(Rogério, Lima)
(Rachel, McAdams)
(Ryan, Gosling)
(Reese, Witherspoon)
(Rafael, Antunes)
(Raquel, Moura)
(Rui, Nogueira)
(Robert, Downey)
(Ricardo, Mendes)
(Renata, Teixeira)
(Sandra, Bullock)
(Sérgio, Oliveira)
(Sara, Marques)
(Sofia, Monteiro)
(Scarlett, Johansson)
(Simone, Oliveira)
(Stephen, King)
(Sophia, Andresen)
(Tatiana, Vieira)
(Tiago, Pereira)
(Tom, Hanks)
(Toni, Morrison)
(Viviane, Lima)
(Victor, Hugo)
(Virginia, Woolf)
(Will, Smith)
(William, Faulkner)
(Zadie, Smith)
(Amber, Heard)
(Anderson, Moura)
(Amanda, Santos)
(Alexandra, Barros)
(Anne, Hathaway)
(Angelina, Jolie)
(Amy, Adams)
(André, Monteiro)
(Ana, Martins)
(Alexandre, Rodrigues)
(Adriana, Lopes)
(Alberto, Pereira)
(Albert, Camus)
(Aldous, Huxley)
(Agatha, Christie)
(Ben, Affleck)
(Brad, Pitt)
(Chadwick, Boseman)
(Carrie, Moss)
(Carolina, Gomes)
(Cláudia, Barbosa)
(Camila, Ribeiro)
(Christian, Bale)
(Carla, Lopes)
(Chris, Hemsworth)
(Clara, Sousa)
(Cristina, Sousa)
(Carlos, Ramos)
(Chinua, Achebe)
(C.S., Lewis)
(Charles, Dickens)
(Clarice, Lispector)
(Douglas, Ramos)
(Diego, Marques)
(Daniel, Radcliffe)
(Diana, Rocha)
(Denzel, Washington)
(Eduarda, Silva)
(Emma, Watson)
(Ernest, Hemingway)
(Edgar, Allan)
(Emily, Dickinson)
(Fábio, Castro)
(Francisco, Ferreira)
(Fabiana, Alves)
(Fyodor, Dostoevsky)
(Franz, Kafka)
(Fernando, Gomes)
(Gal, Gadot)
(Gisele, Ferreira)
(Gonçalo, Cruz)
(Gustavo, Lima)
(George, Orwell)
(Gabriel, Marquez)
(Hugh, Jackman)
(Hugo, Pires)
(Helena, Bonham)
(Haruki, Murakami)
(Herman, Melville)
(Inês, Mendes)
(Isabel, Allende)
(Italo, Calvino)
(Jason, Momoa)
(Júnior, Bastos)
(Juliano, Rocha)
(Jéssica, Costa)
(Joana, Almeida)
(Joaquin, Phoenix)
(Johnny, Depp)
(Julia, Roberts)
(João, Pinto)
(James, Joyce)
(J.R.R., Tolkien)
(J.K., Rowling)
(Jane, Austen)
(José, Martins)
(Jorge, Luis)
(Keanu, Reeves)
(Kelly, Neves)
(Kate, Winslet)
(Lupita, Nyong)
(Larissa, Amorim)
(Luciana, Nogueira)
(Laura, Gomes)
(Leonardo, DiCaprio)
(Liam, Nelson)
(Leo, Tolstoy)
(Luís, Carvalho)
(Márcio, Coelho)
(Matt, Damon)
(Morgan, Freeman)
(Marco, Ferreira)
(Margot, Robbie)
(Meryl, Streep)
(Marta, Ribeiro)
(Maria, Costa)
(Margarida, Maggie)
(Mariana, Matos)
(Marcel, Proust)
(Mark, Twain)
(Miguel, Pires)
(Nicole, Kidman)
(Natalie, Portman)
(Nicolas, Mendes)
(Oscar, Wilde)
(Patrícia, Barros)
(Pedro, Santos)
(Paulo, Carvalho)
(Pablo, Neruda)
(Rodrigo, Pinto)
(Rogério, Lima)
(Rachel, McAdams)
(Ryan, Gosling)
(Reese, Witherspoon)
(Rafael, Antunes)
(Raquel, Moura)
(Rui, Nogueira)
(Robert, Downey)
(Ricardo, Mendes)
(Renata, Teixeira)
(Sandra, Bullock)
(Sérgio, Oliveira)
(Sara, Marques)
(Sofia, Monteiro)
(Scarlett, Johansson)
(Simone, Oliveira)
(Stephen, King)
(Sophia, Andresen)
(Tatiana, Vieira)
(Tiago, Pereira)
(Tom, Hanks)
(Toni, Morrison)
(Viviane, Lima)
(Victor, Hugo)
(Virginia, Woolf)
(Will, Smith)
(William, Faulkner)
(Zadie, Smith)
